flaml/tune/tune.py

```diff
@@ -735,9 +737,14 @@ def easy_objective(config):
                         )
                         results = None
                         with PySparkOvertimeMonitor(time_start, time_budget_s, force_cancel, parallel=parallel):
-                            results = parallel(
-                                delayed(evaluation_function)(trial_to_run.config) for trial_to_run in trials_to_run
-                            )
+                            try:
+                                results = parallel(
+                                    delayed(evaluation_function)(trial_to_run.config) for trial_to_run in trials_to_run
+                                )
+                            except Exception as e:

```

Comment:
We should use a more specific exception.

Reply:
Updated.